tests/test_stamp.py

```diff
@@ -449,6 +451,126 @@ def new_toplevel_only(self, object_types):
+        this_config_str = "" + config_str  # Make a copy
+        if airmass is not None:
+            this_config_str += f"image.bandpass.airmass: {airmass}\n"
+        if camera is not None:
+            this_config_str += f"image.bandpass.camera: {camera}\n"
+            this_config_str += f"image.bandpass.det_name: {det_name}\n"
+
+        config = yaml.safe_load(this_config_str)
+        # If tests aren't run from test directory, need this:
+        config['input.sky_catalog.file_name'] = str(DATA_DIR / "sky_cat_9683.yaml")
+        config['input.opsim_data.file_name'] = str(DATA_DIR / "small_opsim_9683.db")
+        os.environ['SIMS_SED_LIBRARY_DIR'] = str(DATA_DIR / "test_sed_library")
+
+        galsim.config.ProcessAllTemplates(config)
+
+        # Hotfix indeterminism in skyCatalogs 1.6.0.
+        # cf. https://github.com/LSSTDESC/skyCatalogs/pull/62
+        # Remove this bit once we are dependent on a version that includes the above PR.
+        orig_toplevel_only = imsim.skycat.skyCatalogs.SkyCatalog.toplevel_only
+        def new_toplevel_only(self, object_types):
+            return sorted(orig_toplevel_only(self, object_types))
+        imsim.skycat.skyCatalogs.SkyCatalog.toplevel_only = new_toplevel_only
+
+        # Run through some setup things that BuildImage normally does for us.
+        logger = galsim.config.LoggerWrapper(None)
+        builder = galsim.config.valid_image_types['LSST_Image']
+
+        galsim.config.ProcessInput(config, logger)
+        galsim.config.SetupConfigImageNum(config, 0, 0, logger)
+        xsize, ysize = builder.setup(
+            config['image'], config, 0, 0, galsim.config.image_ignore, logger
+        )
+        galsim.config.SetupConfigImageSize(config, xsize, ysize, logger)
+        galsim.config.SetupInputsForImage(config, logger)
+        galsim.config.SetupExtraOutputsForImage(config, logger)
+        config['bandpass'] = builder.buildBandpass(config['image'], config, 0, 0, logger)
+        config['sensor'] = builder.buildSensor(config['image'], config, 0, 0, logger)
+
+        nobj = builder.getNObj(config['image'], config, 0)
+
+        ref_fluxes = []
+        realized_fluxes = []
+        for obj_num in [75, 2697, 2619, 2699, 2538]:  # bright, big galaxy, extremely bright star
+            print(f"{obj_num = }")
+            image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False)
+            realized_flux = config['realized_flux']
+            print(f"{realized_flux = }")
+            gal = galsim.config.BuildGSObject(config, 'gal', logger=logger)[0]
+            ref_flux = gal.sed.calculateFlux(config['bandpass'])
+            print(f"{ref_flux = }")
+            print()
+
+            ref_fluxes.append(ref_flux)
+            realized_fluxes.append(realized_flux)
+
+        print("\n"*3)
+        return np.array(ref_fluxes), np.array(realized_fluxes)
+
+    ref_QE, realized_QE = get_fluxes(None, 'LsstComCam', 'R22_S11')
```

Comment:
Could also just use `LsstCam` here and since those data are already in `obs_lsst_data`.  Note that DM-40164--the ticket with the updated QE curves for LSSTCam-- still isn't merged, as of this writing, but will be soon.

Reply:
Of course.  Hopefully that'll get this through CI.

Reply:
> Hopefully that'll get this through CI.

Nope, not quite.  test_object_positions refers to ComCam and now fails b/c I added `camera` to imsim-config.yaml.  But getting closer...

Reply:
Fiddled with test_object_positions to not use non-existent LsstCamImSim QE curves.